src/main/scala/io/sdkman/changelogs/ActiveMQMigrations.scala

```diff
@@ -6,79 +6,50 @@ import org.bson.Document
 
 @ChangeLog(order = "060")
 class ActiveMQMigrations {
+
+  val CandidateName = "activemq"
+
   @ChangeSet(
     order = "001",
     id = "001-add_activemq_candidate",
     author = "xshyamx"
   )
   def migration001(implicit db: MongoDatabase): Candidate = {
     Candidate(
-      candidate = "activemq",
+      candidate = CandidateName,
       name = "Apache ActiveMQ (Classic)",
       description =
         "Apache ActiveMQ® is the most popular open source, multi-protocol, Java-based message broker. It supports industry standard protocols so users get the benefits of client choices across a broad range of languages and platforms. Connect from clients written in JavaScript, C, C++, Python, .Net, and more. Integrate your multi-platform applications using the ubiquitous AMQP protocol. Exchange messages between your web applications using STOMP over websockets. Manage your IoT devices using MQTT. Support your existing JMS infrastructure and beyond. ActiveMQ offers the power and flexibility to support any messaging use-case.",
       websiteUrl = "https://activemq.apache.org/"
     ).insert()
   }
-
   @ChangeSet(
-    order = "002",
-    id = "002-add_correct_activemq_candidate_urls",
+    order = "003",
+    id = "003-add_new_activemq_versions_and_update_default",
     author = "xshyamx"
   )
-  def migration002(implicit db: MongoDatabase): Document = {
+  def migration003(implicit db: MongoDatabase): Document = {
     List(
+      "5.17.1",
+      "5.17.0",
+      "5.16.5",
+      "5.16.4",
+      "5.16.3",
       "5.16.2",
       "5.16.1",
```

Comment:
It seems like 5.16.1 and 5.16.2 were already in the previous migration.
Are they missing from the database now?

Reply:
I just checked, and they are in the database. Can you remove these two lines, please?

Reply:
Good catch. I also included 5.16.0 as a duplicate.